pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
 requires-python = "==3.8.*"
 dependencies = [
     "transformers==4.44.2",
-    "torch==2.0.0+cpu",
+    "torch==2.3.0+cpu",
```

Comment:
Gemma2 uses torch._dynamo in its init function
`torch._dynamo.mark_static_address(new_layer_key_cache)`
Since dynamo is available torch 2.3 onwards, had to include it for passing the testcases. Otherwise the model still functions.
`FAILED tests/transformers/test_transformer_pytorch_transforms.py::test_kv_cache_transform[gemma2-1-8-2304-kwargs43-0.8] - AttributeError: module 'torch._dynamo' has no attribute 'mark_static_address`